sdk/documentintelligence/azure-ai-documentintelligence/azure/ai/documentintelligence/_operations/_operations.py

```diff
@@ -571,6 +572,730 @@ def begin_analyze_document(
          with MutableMapping
         :rtype: ~azure.core.polling.LROPoller[~azure.ai.documentintelligence.models.AnalyzeResult]
         :raises ~azure.core.exceptions.HttpResponseError:
+
+        Example:
+            .. code-block:: python
+
+                # JSON input template you can fill out and use as your body input.
+                analyze_request = {
+                    "base64Source": bytes("bytes", encoding="utf-8"),  # Optional. Base64
+                      encoding of the document to analyze.  Either urlSource or base64Source must be
+                      specified.
+                    "urlSource": "str"  # Optional. Document URL to analyze.  Either urlSource or
+                      base64Source must be specified.
```

Comment:
@iscai-msft document intelligence has models, should it be generating with the request/response templates in the docstrings?

Reply:
Let's follow up with Izzy offline about this

Reply:
We had decided that we were going to cc @johanste and @lmazuel, because we want to generate with sample input. We will probably change what our sample looks like in the future, but we do want to expose sample input to users